### DISCUSSION TITLE
Email Address To be Updated

### DISCUSSION THREAD

**Discussion by christophercutajar** [argoproj/argo-events#1340] [General]
In the [security policy](https://github.com/argoproj/argo-events/security/policy), it is advised that security vulnerabilities are to be sent to:

```
* [Jesse_Suen@intuit.com](mailto:Jesse_Suen@intuit.com)
* [Alex_Collins@intuit.com](mailto:Alex_Collins@intuit.com)
* [Edward_Lee@intuit.com](mailto:Edward_Lee@intuit.com)
```

After sending an email to the aforementioned email addresses, I got an address not found email from the `Mail Delivery Subsystem`.

![image](https://user-images.githubusercontent.com/60098546/132380584-fd5ddae2-3106-4014-b25a-65dea5b44a47.png)

I think the policy needs to be updated.

**Comment by whynowy** [comment]
done.
